qa/common/deploy.sh

```diff
@@ -136,6 +136,17 @@ function initialization_sequence {
     set -x
 }
 
+function salt_api_test {
+    local tmpfile=$(mktemp)
+    echo "Salt API test: BEGIN"
+    systemctl --no-pager --full status salt-api.service
+    curl http://$(hostname):8000/ | tee $tmpfile
+    test -s $tmpfile
+    jq . $tmpfile
+    rm $tmpfile
```

Comment:
So a simple `curl http://$(hostname):8000/ | jq` does not work, and I don't know why, exactly. This is the workaround I came up with - I like it because it shows the raw curl output. Before we were piping that to `python` so it was not shown in the log.

Reply:
@smithfarm the error: `(23) Failed writing body` is because of jq differently handles buffers at some reason, I found workaround, so jq can work properly. Just put the minimal filter exclusively, i. e. to use `jq .`. In case you are interested how to reproduce this try next commands:
- works: 
```echo '["hello", "world"]' | jq```
- fails:
```echo '["hello", "world"]' | jq | cat```
- works again 
```echo '["hello", "world"]' | jq . | cat```